src/client/eppo-client.ts

```diff
@@ -988,6 +996,32 @@ export default class EppoClient {
     validateNotBlank(flagKey, 'Invalid argument: flagKey cannot be blank');
 
     const flagEvaluationDetailsBuilder = this.newFlagEvaluationDetailsBuilder(flagKey);
+    const overrideVariation = this.overridesStore?.get(flagKey);
+    if (overrideVariation) {
+      const configFormat = this.overridesStore?.getFormat() ?? '';
+      const flagEvaluationDetails = flagEvaluationDetailsBuilder
+        .setMatch(
+          0,
+          overrideVariation,
+          { key: overrideVariation.key, splits: [], doLog: false },
```

Comment:
the value for `key` may be confusing, I wonder if instead we make `allocation` nullable and leave this null when override applied. Or maybe make the value of `key` `override-<variation key>` so it's obvious.